### PR TITLE
fix(placement): adjust hover style for links

### PR DESCRIPTION
**Pull request by caugner**
### Problem

The "Mozilla ads" and "Don't want to see ads?" are not always recognizable as links, because they don't always change their `text-decoration` on hover.

### Solution

Make sure that the `text-decoration` changes on hover.

---

### Before

https://github.com/mdn/yari/assets/495429/757b7cfe-0053-416a-b32d-5128aae9d5cb

### After

https://github.com/mdn/yari/assets/495429/4f99a8d0-614b-4a1c-9262-2e478a2e117f

---

## How did you test this change?

Ran `yarn dev` with `REACT_APP_KUMA_HOST=developer.allizom.org` and `REACT_APP_PLACEMENT_ENABLED=true` in my `.env` and looked at http://localhost:3000/en-US/docs/Web/HTML, then compared with `main`.

### CODE REVIEW

**Review by LeoMcA — APPROVED** [review]
code lgtm, I'll let you test when you add screenshots :rocket:

**Review by LeoMcA — APPROVED** [review]
Code looks good, but a super opinionated view (so feel free to ignore) on the "Mozilla ads" underline (I think it looks bad):

File: client/src/ui/organisms/placement/index.scss [hunk 1]
```diff
@@ -47,6 +47,21 @@ section.place {
     }
   }
 
+  .pong-note {
+    text-decoration: none;
+
+    &:active,
+    &:link,
+    &:visited {
+      text-decoration: underline;
```
[the remark this answers is not in the document]
Comment (caugner) [reply client/src/ui/organisms/placement/index.scss:56]: Oh, that change was indeed not intended. I just realise that the current text-decoration is different between top and sidebar banner. I missed that today, and will provide a fix.

**Review by caugner — COMMENTED** [review]
Self-approval, and tested again:

1. In the top, both "Mozilla ads" and "Don't want to see ads?" are underlined only on hover.
2. In the sidebar, "Mozilla ads" is underlined except on hover, and "Don't want to see ads?" is underlined only on hover.
3. In the footer, "Mozilla ads" is underlined except on hover, and "Don't want to see ads?" is underlined only on hover.

Bottom line is this now works as expected, and code changes have already been reviewed by Leo.

**Pull request merged**